libs/ui/widgets/src/lib/permalink/permalink.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="flex">
+  <input
+    type="text"
+    class="grow py-2 px-4 rounded-l-md border border-gray-300"
+    value="{{ link }}"
+    readonly
+  />
+  <button
+    type="button"
+    class="py-2 px-4 rounded-r-md bg-gray-300 hover:bg-gray-500 focus:bg-gray-700 text-white"
+    (click)="copyPermalink($event)"
+  >
+    {{ 'permalink.button.copy' | translate }}
+  </button>
+</div>
```

Comment:
I feel like this could be generalized to something like a "copyable text field". This is also what we will use for the web components snippet, and we could use it in any place where a "copy" button is offered (for example by offering an option to show/hide the text to copy). This is typically the case on the API links in the datahub.

In the end, it is not really specific to this permalink.

Reply:
Absolutely agree and adapted accordingly, moving it to the `inputs` lib as well